src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravellerSelection.tsx

```diff
@@ -66,7 +66,8 @@ export function TravellerSelection({
   >(selectableUserProfiles);
 
   const canSelectUserProfile = !(
-    selectionMode === `single` && selectableUserProfiles.length <= 1
+    (selectionMode === `single` || selectionMode === 'none') &&
+    selectableUserProfiles.length <= 1
```

Comment:
The logic in the purchase screen is slightly more precise:

```tsx
const canSelectUserProfile = !(
  travellerSelectionMode === `none` ||
  (travellerSelectionMode === `single` && selectableTravellers.length <= 1)
)
```

Maybe drag it out to a util-function?